omod-common/src/test/java/org/openmrs/module/webservices/rest/web/api/impl/RestServiceImplTest.java

```diff
@@ -340,197 +340,6 @@ public void getResourceByName_shouldReturnResourceWithLowerOrderValueIfTwoResour
 	}
 	
 	/**
-	 * @see RestServiceImpl#getSearchHandler(String,Map)
-	 * @verifies throw exception if no handler with id
-	 */
-	@Test
-	public void getSearchHandler_shouldThrowExceptionIfNoHandlerWithId() throws Exception {
-		
-		SearchHandler searchHandler = mock(SearchHandler.class);
-		SearchConfig searchConfig = new SearchConfig("none", "concept", "1.8.*", new SearchQuery.Builder("Fuzzy search")
-		        .withRequiredParameters("q").build());
-		when(searchHandler.getSearchConfig()).thenReturn(searchConfig);
-		when(restHelperService.getRegisteredSearchHandlers()).thenReturn(Arrays.asList(searchHandler));
-		
```

Comment:
yes, I moved the tests for getSearchHandler down so tests follow the same order as methods in RestServiceImpl